Scenes/LoginScreen.cs

```diff
@@ -1,20 +1,44 @@
 using Godot;
+using System;
 
 public class LoginScreen : Control
 {
   private Authorization authorization;
-
+  private LineEdit textField;
+  Boolean activateAuthDev = false;
   public Serilog.Core.Logger _serilogger;
 
   [Signal] public delegate void loginSuccess(string userId);
 
   public override void _Ready()
   {
-    authorization = authorization = new Authorization();
-    this.AddChild(authorization);
+    var clientConfig = new ConfigFile();
+    Godot.Error err = clientConfig.Load("res://Resources/client.cfg");
+
+    if (err == Godot.Error.Ok)
+    {
+      // enable/disable authentication in dev mode
+      activateAuthDev = (Boolean)clientConfig.GetValue("auth", "activate_auth_dev");
```

Comment:
In the case where there's no config file, we need defaults baked in to the code. When we do a build of the client and distribute it, the config file doesn't get baked in. The config file really only gets used for dev.

Reply:
I still have to test these new changes.

Since the configuration values do not change at runtime, it may be worthwhile to create a static class that can be accessed from anywhere in the game, and initialize it in the Main game. This will make it easier to manage and modify the configuration values in the future

Reply:
You're welcome to do that. The server just embeds the defaults into the places where the config file is loaded.